io.elementary.Loki.BaseApp.json

```diff
@@ -0,0 +1,84 @@
+{
+    "id": "io.elementary.Loki.BaseApp",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "separate-locales": false,
+    "cleanup": [
+        "/cache",
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/aclocal",
+        "/share/devhelp",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "/lib/systemd",
+        "*.la", "*.a"
+    ],
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+
+    "modules": [{
+            "name": "vala",
+            "cleanup-platform": ["*"],
+            "config-opts": [
+                "--enable-vapigen", "--enable-unversioned"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/vala/0.36/vala-0.36.14.tar.xz",
+                "sha256": "31df542f37f78d8847c3675a76f691df9e8a6ccfd93045ca3a33cd25db3aa4d2"
+            }]
+        },
+        {
+            "name": "libgee",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.0.tar.xz",
+                "sha256": "21308ba3ed77646dda2e724c0e8d5a2f8d101fb05e078975a532d7887223c2bb"
+            }]
+        },
+        {
+            "name": "granite",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Is there a releasewithdebinfo option? The debug info should be stripped by flatpak into a .Debug extension but it'd likely be helpful for developers to have this enabled.

(note: this is a suggestion not a requirement)